CefSharp.Example/Resources/BindingTest.html

```diff
@@ -113,6 +113,32 @@
         </p>
 
         <p>
+            Disposed Javscript Callback Test (navigates to www.google.com before callback fires)
```

Comment:
How do I split it out into its own test case?  Is there any guidance or
some other test case I can look at?

On Sun, Oct 18, 2015 at 4:16 PM, Alex Maitland <notifications@github.com>
wrote:

> In CefSharp.Example/Resources/BindingTest.html
> <https://github.com/cefsharp/CefSharp/pull/1344#discussion_r42328223>:
>
> > @@ -113,6 +113,32 @@
> >          </p>
> >
> >          <p>
> > +            Disposed Javscript Callback Test (navigates to www.google.com before callback fires)
>
> Need to split this out into it's own test case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cefsharp/CefSharp/pull/1344/files#r42328223>.
>